source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButton.mm

```diff
@@ -10,23 +10,24 @@
 #import "ACRView.h"
 
 @implementation UIButton(ACRButton)
+-(instancetype)initWithCoder:(NSCoder *)aDecoder
+{
+    self = [super initWithCoder:aDecoder];
+    return self;
+}
+
```

Comment:
You can remove this method, since it's just repeating the superclass's implementation.